hypermap/aggregator/tests.py

```diff
@@ -68,3 +70,43 @@ def test_pages_render(self):
         response = self.client.get(
             reverse('layer_detail', args=(str(layer.id),)))
         self.assertEqual(200, response.status_code)
+
+    def test_update_layers_esri(self):
+        arcservice = ArcMapService("http://sampleserver1.arcgisonline.com/ArcGIS/rest/services"
```

Comment:
I am not sure it is wise to run a test with an external dependency, because it the service is not running it will make our test suit fail even if our code is in good health.
Maybe check before running the test if the url return a valid json response, and exit in case not?
@ingenieroariel and @simod what is your opinion?